src/app/auth/routes.py

```
@@ -3,15 +3,27 @@
 from __future__ import annotations
 
 import logging
+from os import error
```

Flake8 (7.3.0)

[error] 'os.error' imported but unused

(F401)